### ISSUE TITLE
Text for Multiple File Imput

### ISSUE THREAD

**Issue by danielhickman** [Dogfalo/materialize#1787]
The file input's text box only shows the first file selected rather than the whole set.

```html
<form action="didlysquat.php">
<div class="file-field input-field">
  <input class="file-path validate" type="text"/>
  <div class="btn">
    <span>Upload Files</span>
    <input type="file" name="filesToUpload[]" id="filesToUpload" multiple/>
  </div>
</div>
</form>
```

My proposed fix:
Use a similar function to this:
```js
var fileFormID = document.getElementById('fileFormID');
for (var i = 0; i < fileFormID.files.length; ++i) {
// append: fileFormID.files.item(i).name; into an array
}
```
Instead of `document.getElementById('fileFormID').value`

**Comment by carlosperate** [comment]
Sounds like a good fix, could you provide a pull request?

**Comment by acburst** [comment]
This was merged into master #1654 a bit ago. Its not in the release version but if you are up to date with master it is there.
